sparse/boot/flash-on-windows.bat

```diff
@@ -138,7 +138,7 @@ echo %oemblobwebsite%
 echo(
 echo Ensure you download %oemblobversion% of the image, which can be found under:
 echo "Software binaries for AOSP Oreo (Android 8.1) - Kernel 4.4 - Nile"
-echo either (latest) or (%oemblobversion%) and unzip it into this directory.
+echo either ^(latest^) or ^(%oemblobversion%^) and unzip it into this directory.

```

Comment:
why not in quotemarks like the line above?

Reply:
Welcome to Windows .bat universe :grinning: I was caught out with this myself when looking quickly.
`echo "Hello world"` in Windows outputs:
```"Hello world"```

That `"Software binaries for AOSP Oreo..."` was originally meant to be between the quotemarks, as it is a title in the webpage :slightly_smiling_face:

Reply:
To add to the madness, guess what the output of `echo "Hello world^(?^)"` is :laughing: 
Hint: it's not `echo "Hello world(?)"`